tests/h/groups/views_test.py

```diff
@@ -544,6 +562,61 @@ def test_leave_publishes_leave_event(Group, session_model):
     })
 
 
+def annotation(document, groupid=u'__world__', shared=False):
+    """
+    Add an annotation of the given Document to the db and return it.
+
+    The given Document must already have an associated DocumentURI.
+
+    """
+    annotation = models.Annotation(
+        userid=u'fred', groupid=groupid, shared=shared,
+        target_uri=document.document_uris[0].uri)
+    db.Session.add(annotation)
+    return annotation
+
+
+def document(db, uri):
```

Comment:
Nit: Might be nice to have the db session similar to the `annotation` helper function, or the other way around.